protos/generate.sh

```diff
@@ -1,14 +1,23 @@
 #!/usr/bin/env bash
 set -e
 
-GIT_ROOT=$(git rev-parse --show-toplevel)
-cd "$GIT_ROOT"
 
-# Install python grpcio-tools if it's not already installed
-# pip install grpcio-tools
+if [ ! -d "${PWD}/.git" ]; then
```

Comment:
the goal of `git rev-parse --show-toplevel` is to allow running it from sub directories of the git repo, this check is not really necessary

Reply:
it ok to not support running this from outside of the bentoml git repository